packages/zoe/src/contracts/loan/borrow.js

```diff
@@ -48,16 +56,16 @@ export const makeBorrowInvitation = (zcf, config) => {
 
     const collateralPriceInLoanBrand = quoteAmount.value[0].amountOut;
 
-    // formula: assert collateralValue*100 >= loanWanted*mmr
+    // formula: assert collateralValue*100 >= loanWanted*mmrRatio
 
     // Calculate approximate value just for the error message if needed
-    const approxForMsg = mmr.scale(loanWanted);
+    const approxForMsg = multiplyBy(loanWanted, mmrRatio);
 
     // Assert the required collateral was escrowed.
     assert(
       loanMath.isGTE(
         loanMath.make(collateralPriceInLoanBrand.value),
```

Comment:
This didn't change in this PR, but I just realized this line doesn't make any sense. we are remaking an amount from an amount.